### DISCUSSION TITLE
Feature Request: multiple button rows

### DISCUSSION THREAD

**Discussion by Justalurker1213** [Clooos/Bubble-Card#280] [Feature requests]
It would be nice to have the option for more than one row of popup buttons rather than 1 row that you swipe.
I tried creating a vertical stack card and splitting the buttons that way but the formatting does not seem to allow this.

**Comment by lazyboy0172** [comment]
I think this could be implemented by adding more options for the 'button' card type. Rather than just forcing a toggle of a switch or a slider for lights/media, allow configurable tap_action and hold_actions, so a user could create their own dashboard looking they way they want with bubble 'button' cards, and configure them to navigate to popup cards, similar to the 'regular' button card in lovelace.
